### ISSUE TITLE
[Bug] [Addon] Official YouTube addon doesn't seem to work

### ISSUE THREAD

**Issue by bradenbest** [Stremio/stremio-bugs#980]
**Describe the bug**
The official youtube addon is either broken or useless, as I have tried multiple times to use it to watch literally anything on youtube, and the only thing I managed to get playing was a random movie trailer which I wasn't even looking for in the context of youtube. I even tried searching super popular channels (like markiplier and pewdiepie), and pasting direct youtube urls into the search bar, to no avail. If it's this hard to pull up a youtube video on stremio, why would I bother with it when it takes 5 seconds to pull it up on the actual website? The addon touts an ad-free experience, but I can also get this by using an ad blocker at any level of my network or downloading.

**To Reproduce**
1. Install the official youtube addon
2. Try to search for literally any youtube video

**Expected behavior**
I would expect the youtube addon to provide some kind of frontend to youtube, including a youtube section in the search results that is as functional as youtube's own search frontend, and, at the bare minimum, the ability to watch arbitrary youtube videos

**Screenshots**
![image](https://github.com/user-attachments/assets/86b813bb-a5b5-4924-b3cc-17b5994c2d1c)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser N/A (desktop app)
 - Version 4.4.168

**Smartphone (please complete the following information):**
N/A

**TV (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by bradenbest** [comment]
I did find a channels feed in the discovery tab, but this is also not ideal, since it relies on the channels being listed there. I had to scroll pretty far down to find markiplier's channel and it did actually play a video from his channel and provide a search box fro me to search his videos (although the latest videos listed were from over a year ago). But therein lies the problem: Markiplier and pewdiepie were just examples, because I know they are mega popular and a good way to test if it works at all. But what about the channels I actually watch? How long would I have to scroll to get to vewn or lockpickinglawyer? Are they even listed? Assuming I scrolled through this endless feed for 18 hours and actually managed to find a channel I'm subscribed to, would any of their videos actually be listed or are they "too obscure"? Even Markiplier's recents only go up to last year, implying there's some kind of manual curation going on here. 

Which begs the question: why would one use this over the website proper?
